js/src/components/experience-rating-banner/feedback-modal.js

```diff
@@ -10,6 +10,8 @@ import { external as externalIcon } from '@wordpress/icons';
  */
 import AppModal from '../app-modal';
 import AppButton from '../app-button';
+import { APP_RATINGS_BANNER_CONTEXT } from '~/constants';
+import { recordGlaEvent } from '~/utils/tracks';
 
 /**
  * FeedbackModal component.
```

Comment:
@mukeshpanchal27 This JSDoc must be before the component and after all the `@event` docs. Can you please update it to add the `@fires` property?